ResearchKit/Common/ORKPasscodeViewController.h

```diff
@@ -129,6 +129,17 @@ ORK_CLASS_AVAILABLE
  */
 + (BOOL)removePasscodeFromKeychain;
 
+/**
+ This method allows to force the passcode to be a certain value. You only want to use this capability when you're moving your existing users
+ from a custom passcode entry mechanism to the one provided by ResearchKit.
+ 
+ This method will raise an exception if `passcode` is nil.
+ 
+ @param passcode          The passcode to store
+ @param touchIdEnabled    Whether TouchId will be available during passcode entry
+ */
++ (void)forcePasscode:(NSString *)passcode withTouchIdEnabled:(BOOL)touchIdEnabled;
```

Comment:
I chose to make this method public because if you use ORKKeychainWrapper directly, you need to know several internal keys. This is how the passcode gets stored, showing the three keys `KeychainDictionaryPasscodeKey `, `KeychainDictionaryTouchIdKey` and `forKey:PasscodeKey ` that one must use correctly:

```obj-c
NSDictionary *dictionary = @{
    KeychainDictionaryPasscodeKey : [passcode copy],
    KeychainDictionaryTouchIdKey : @(touchIdEnabled)
};
NSError *error;
[ORKKeychainWrapper setObject:dictionary forKey:PasscodeKey error:&error];
```

We could think of making a public method on the keychain wrapper, but since these keys live inside `ORKPasscodeStepViewController_Internal.h` we'd need to move those as well. Making it a class-level public method on the passcode (step) view controllers seemed like the least intrusive way to me, but I'm certainly open to other solutions.